interview-questions.md

```diff
@@ -7,53 +7,54 @@ Then, research the question to expand on your answer. Even if you feel you have
 
```

Comment:
I understand you were meeting a time crunch. These interview questions need a bit more content. Please be consistent with your approach on the questions, even when it feels like you can answer it quickly. It is not a speed race. It is content paced. Show that interviewer that you understand the tool and have used it.